frame/utility/src/lib.rs

```diff
@@ -149,7 +149,7 @@ pub mod pallet {
 		})]
 		pub fn batch(
 			origin: OriginFor<T>,
-			calls: Vec<<T as Config>::Call>,
+			calls: Vec<Box<<T as Config>::Call>>,
```

Comment:
This doesn't make any sense. When you use a Vec, these values are already "boxed". What you want to achieve here?

Reply:
inside a vec, the array of element is boxed, not each element individually.
because Call is around 400B, for a `Vec<Call>` of capacity 10 allocate an array of size 10*400 B.
whereas a `Vec<Box<Call>>` of capacity 10 allocate an array of size `10*mem::size_of<usize>() B` and also allocate 10 individual calls.
This way the maximum length of an individual allocation is minimized.

Reply:
Yeah that is right, but what is the reason for doing this? 

We will not have a call vec with 83886 calls anyway?